content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -98,8 +130,96 @@ aws configure list
+4. On the Set Group Name page, for Group Name, enter a name for the new group. 
+5. Choose Next Step.
+6. On the **Attach Policy** page, choose **Next Step** without attaching any policies or attach any current policies relevant to the group.
+7. Choose **Create Group**. 
+
+#### Create a user group via the AWS CLI
+1. Install and configure the AWS CLI as above, see [Installing the AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/installing.html).
+2. Generate a user group
+```cmd
+aws iam create-group --group-name MyUserGroup
+```
+
+#### Create user groups via the AWS CDK Examples
+The AWSCore CDK stack will autogenerate sample Admins and Users user groups for you. By default, these groups are named ```O3DE-AWS-PROJECT-Admins``` and ```O3DE-AWS-PROJECT-Users```.
+
+See the [CDK setup instructions](cdk-application) for more details.
+
+### Add Users to the User Group
+You can add users to IAM User Groups via the console or the CLI. See [Adding and removing users in an IAM user group ](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups_manage_add-remove-users.html) for full details.
+
+You can quickly add users via the cli as follows:
+
+```
+aws iam add-user-to-group --group-name MyGroup --user-name MyNewUser
+```
+
+### Attach Permissions to the User Group
+You can attach IAM policies to a user group to control the permissions they have access to. This can be done via the AWS Console, the AWS CLI or via CloudFormation/CDK. See [Attaching a policy to an IAM user group ](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups_manage_attach-policy.html) to attach permissions via the AWS Console or CLI.
+
+#### Working with O3DE CDK Examples
+If you deploy both the Core and example stack from the AWS Core Gem you can see examples of automatically generated user permissions via the ```generate_user_policy``` and ```generate_admin_policy``` functions, which are then automatically attached to the user groups via the call to ```__grant_access```.
+
+See the CDK [Permissions](https://docs.aws.amazon.com/cdk/latest/guide/permissions.html) documentation for more details.
+
+The AWS feature gems, such as AWS Metrics Gem, during deployment automatically create managed policies for users and admins that can then be attached manually to the appropriate user group. 
+
+##### Attach Managed Policies to User Groups via the AWS Console
+* Open the AWS console at https://console.aws.amazon.com/cloudformation.
+* Navigate to the desired stack in a region.
+* Click the **Resources** tab.
+* Look for resources with the type ```AWS::IAM::ManagedPolicy``` in the stack and record the policy name.
+* Next go to IAM's console at https://console.aws.amazon.com/iam
+* Select **user groups**
+* Click on the user group
+* Select **permissions**, then **add permissions** and select **attach polices**.
+* Filter for policy to attach and select it
+* Hit **Attach Policies**.
+
+##### Attach Managed Policies to User Groups via the AWS CLI
```

Comment:
This should be a numbered (ordered) list, not a bulleted list.
Also, each command line example should be in an indented code block immediately following the step.

Reply:
Fixed